gradle/validation/jar-checks.gradle

```
@@ -200,6 +200,10 @@ subprojects {
     description = "Validate license and notice files of dependencies"
     dependsOn collectJarInfos
 
+    def outputFileName = "validateJarLicenses"
+    inputs.dir(file(project.rootDir.path + "/solr/licenses"))
```

```suggestion
    inputs.dir(file(project.rootDir.path + "/solr/licenses"))
```
is this not using absolue path sensitivity here? 
also let's give this input property a name